### DISCUSSION TITLE
Printing on HP stops for several seconds on each page

### DISCUSSION THREAD

**Discussion by ChristopheFR** [sumatrapdfreader/sumatrapdf#4595] [Q&A]
Hello, I have been using Sumatra for about 5 years. Today I am on Windows 11 with Sumatra 3.5.2 64bits. My printer is HP laserjet pro 500 color MFP. My entire system is up to date.
Since I have been using Sumatra, when I want to print a document of several pages, printing stops for several seconds on each page. Until now I was patient because in 90% of cases, I printed documents of only 1 to 3 pages.
Today my needs have changed and I very often print documents of 70 to 500 pages. I can no longer afford to wait, it is becoming unacceptable.
Yesterday, I tried with ADOBE ACROBAT READER latest version and printing goes well, all the pages are printed one after the other without interruption.

These findings are valid for these 2 applications, whether I use a PCL or PS driver, regardless of double-sided printing or single page printing.

Can you help me correct this problem with Sumatra? I really like this software and I would like not to have to change it.
Thank you.

**Comment by GitHubRulesOK** [comment]
Nothing to correct SumatraPDF as a viewer application hands over to Windows Printing from the time you ask for a print.

However Acrobat (with its light reader) and GhostScript are full printer based applications.

The best inbuilt (native) Windows printing route is use MS Edge with its own inbuilt Acrobat Reader.
